### DISCUSSION TITLE
How to record a hls stream with video only?

### DISCUSSION THREAD

**Discussion by wintbiit** [streamlink/streamlink#5995] [Q&A]
As said, I want drop audio track and save video only for the stream.

**Comment by bastimeyer** [comment]
- https://streamlink.github.io/cli.html#cmdoption-stdout
- https://ffmpeg.org/ffmpeg.html#Stream-selection

```sh
streamlink --stdout ... | ffmpeg -i pipe:0 -an -c:v copy output.mp4
```

**Reply by wintbiit** [reply]
thanks a lot
